packages/examples/cli/README.md

```diff
@@ -0,0 +1,23 @@
+# cli
+
+Run Dockest via the CLI
+
+Specify which services to run using `.dockest` file. Structure:
+
+```
+serviceName,serviceType
+```
+
+For example
+
+```
+my_redis_service,redis
+```
+
+This'll run the default healthchecks for that serviceType.
```

Comment:
How would you set up custom health checks?

Reply:
I didn't address the custom healthchecks in this PR, figured it could be implemented in future iterations of the CLI